app/screens/receive-bitcoin-screen/receive-bitcoin.tsx

```diff
@@ -56,9 +55,26 @@ const styles = EStyleSheet.create({
     color: palette.coolGrey,
   },
 })
-const ReceiveBitcoin = () => {
+const ReceiveBitcoinScreen = ({
+  navigation,
+  route,
+}: StackScreenProps<RootStackParamList, "receiveBitcoin">) => {
+  const { receiveCurrency: initialReceiveCurrency } = route.params || {}
+
   const { usdWalletId } = useMainQuery()
-  const [receiveCurrency, setReceiveCurrency] = useState<CurrencyType>(CurrencyType.BTC)
+  const [receiveCurrency, setReceiveCurrency] = useState<WalletCurrency>(
+    initialReceiveCurrency || WalletCurrency.BTC,
+  )
+
+  useEffect(() => {
+    if (receiveCurrency === WalletCurrency.USD) {
+      navigation.setOptions({ title: translate("ReceiveBitcoinScreen.usdTitle") })
+    }
+
+    if (receiveCurrency === WalletCurrency.BTC) {
+      navigation.setOptions({ title: translate("ReceiveBitcoinScreen.title") })
+    }
+  }, [receiveCurrency])
```

Comment:
There's some eslint warnings being generated here.  Either add the missing properties or disable the rule for this line.